.gitignore

```diff
@@ -1,4 +1,4 @@
 node_modules
 Thumbs.db
 .DS_Store
-jspm_packages
\ No newline at end of file
+demo/dist
```

Comment:
`demo/dist` is in the gitignore here, but the file `demo/dist/index.html` is being checked in.
Should the dist/index.html be checked in?